### DISCUSSION TITLE
Finding Tags in file

### DISCUSSION THREAD

**Discussion by pwbrown44** [vkbo/novelWriter#678] [Help]
Hello,

I have all my tags in the correct place, under the title.  I have 50 tags for one file alone. 

Now, within that one file that has the 50 tags at the top of the file, and over 1,000 words as the text, how do I specifically see where the tagged word is located within the 1,000 words?

Thank you

**Comment by vkbo** [comment]
Do you mean the `@tag:` or one of the reference keywords?
The tags and references only points to the whole section defined by a title, not to individual words. The tags themselves don't show up anywhere until you make a reference to them in a Novel file. Then the cross links will show up in the Outline tab for the whole project, and in the References panel under the document viewer for the file open for viewing.

It doesn't sound like you are using this correctly. Perhaps create a new project with example files (one of the options in the New Project Wizard). That will create a project for you with files that demonstrate how the tags work.

**Reply by pwbrown44** [reply]
Thank you for the prompt response.  I did use the template, and followed the examples. They are great!.  I was hoping to find the word that I used as a tag also within the document.  For example I have the tag [@tag: value] under [object].  Next I would add the tag to the heading of a whole section.  However, I wanted to find the word "value" listed in the section. 

No problem, I love this system! It is making life much easier.

**Reply by vkbo** [reply]
I see (I think). Thanks!

I'll admit that the app is a bit of a programmer's approach to the whole thing, but that's how I'm used to working. I actually started making this as a pure command line tool that processed plain text files. But I'm glad I made it a GUI app.
